clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 #### Fixed
 - Inconsistent completion list with implicitly defined private CSECT
 - Incorrect results of string comparison in CA expressions
+- Incorrect results of CA conversion functions for multibyte UTF-8 codepoints
```

Comment:
Not sure here - is this a fix for an issue in the currently released version or just a fix for the modifications done for the next release?

Reply:
All the modified functions had that upfront length check, which I think was incorrect for the multibyte case.